disk.go

```diff
@@ -23,4 +26,13 @@ type Disk interface {
 
 	// Meta returns the disk metadata.
 	Meta() Meta
+	DiskType() DiskType
```

Comment:
Let's add a comment:
```suggestion

	// DiskType returns either ssd or hdd.
	DiskType() DiskType
```